src/textacy/constants.py

```diff
@@ -21,6 +21,40 @@
 OBJ_DEPS: set[str] = {"attr", "dobj", "dative", "oprd"}
 AUX_DEPS: set[str] = {"aux", "auxpass", "neg"}
 
+"""
+Ordinal points of the token.is_quote characters, matched up by start and end.
+
+source:
+switch = "\"\'"
+start = "“‘```“‘«‹「『„‚"
+end = "”’’’’”’»›」』”’"
+
+"""
+QUOTATION_MARK_PAIRS = {
+    (34, 34),
```

Comment:
not critical, but helpful: could you add the corresponding marks in a comment on each line?